viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if not (title and genre and rating):
+        return None
+    else:
+        new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+        
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)          
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data 
+
+def watch_movie(user_data, movie_title):
+    for movie in user_data["watchlist"]:    
+        if movie_title in movie["title"]:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
+
+    return user_data

```

Comment:
This is a good approach. Modifying a list as it's being iterated over can cause a lot of bugs and errors. Breaking out of the list after line 28 could end the loop and eliminate the possibility of bugs. You could also take the return statement on line 30 and move it over so it's inside of the for loop.